vignettes/endpoints.Rmd

```diff
@@ -0,0 +1,387 @@
+
+Example call:
+
+```{r}
+fluview_clinical(regions = "nat", epiweeks = epirange(201601, 201701)) %>% fetch_tbl()
+```
+### Fluview metadata
+
+Function reference: <https://cmu-delphi.github.io/epidatr/reference/fluview_metadata.html>
+
+Example call:
+
+```{r}
+fluview_meta() %>% fetch_tbl()
+```
+
+TODO broken?
+
+### Google Flu Trends data
+API docs: <https://cmu-delphi.github.io/delphi-epidata/api/gft.html>
+
+Function reference: <https://cmu-delphi.github.io/epidatr/reference/gft.html>
+
+```{r}
+gft(locations = "hhs1", epiweeks = epirange(201201, 202001)) %>% fetch_tbl()
+```
+
+### KCDC ILI
+API docs: <https://cmu-delphi.github.io/delphi-epidata/api/kcdc_ili.html>
+
+Function reference: <https://cmu-delphi.github.io/epidatr/reference/kcdc_ili.html>
+
+kcdc_ili(regions = "01", epiweeks = epirange(199001, 202301)) %>% fetch_tbl()
+TODO example is broken
+
+### NIDSS Flu
+API docs: <https://cmu-delphi.github.io/delphi-epidata/api/nidss_flu.html>
+
+Function reference: <https://cmu-delphi.github.io/epidatr/reference/nidss_flu.html>
+
+```{r}
+nidss_flu(regions = "taipei", epiweeks = epirange(200901, 201301)) %>% fetch_tbl()
+```
+
+### ILI Nearby Nowcast 
+API docs: <https://cmu-delphi.github.io/delphi-epidata/api/nowcast.html>
+
+Function reference: <https://cmu-delphi.github.io/epidatr/reference/nowcast.html>
+
+```{r}
+nowcast(locations = "ca", epiweeks = epirange(202201, 202319)) %>% fetch_tbl()
+```
+
+## Dengue Endpoints
+
+### Delphi's Dengue Nowcast
+
+API docs: <https://cmu-delphi.github.io/delphi-epidata/api/dengue_nowcast.html>
+
+Function reference: <https://cmu-delphi.github.io/epidatr/reference/dengue_nowcast.html>
+
+Example call:
+
+```{r}
+dengue_nowcast(locations = "pr", epiweeks = epirange(201401, 202301)) %>% fetch_tbl()
+```
+
+### NIDSS dengue
+API docs: <https://cmu-delphi.github.io/delphi-epidata/api/nidss_dengue.html>
+
+Function reference: <https://cmu-delphi.github.io/epidatr/reference/nidss_dengue.html>
+
+```{r}
+nidss_dengue(locations = "taipei", epiweeks = epirange(200301, 201301)) %>% fetch_tbl()
+```
+
+## PAHO Dengue
+
+API docs: <https://cmu-delphi.github.io/delphi-epidata/api/paho_dengue.html>
+
+Function reference: <https://cmu-delphi.github.io/epidatr/reference/paho_dengue.html>
+
+```{r, eval=FALSE}
+paho_dengue(regions = "1", epiweeks = epirange(200201, 202319)) %>% fetch_tbl()
+```
+TODO this is disallowed (http 401)
+
+## Other Endpoints
+### Wikipedia Access
+API docs: <https://cmu-delphi.github.io/delphi-epidata/api/wiki.html>
+
+Function reference: <https://cmu-delphi.github.io/epidatr/reference/wiki.html>
+
+```{r}
+wiki(language = "en", articles = "influenza", epiweeks = epirange(202001, 202319)) %>% fetch_tbl()
+```
+# Private methods
+
+These require private access keys to use. To actually run these locally, if you have stored these secrets in `secrets.R` in the root directory of this project, first run
+```{r, eval=FALSE}
+source("../secrets.R")
```

Comment:
You can also [use .Renviron in your local directory](https://support.posit.co/hc/en-us/articles/360047157094-Managing-R-with-Rprofile-Renviron-Rprofile-site-Renviron-site-rsession-conf-and-repos-conf) to set env vars that will be auto-loaded by R.